### DISCUSSION TITLE
Is there a proof that S¹ × S¹ ≃ T²?

### DISCUSSION THREAD

**Discussion by ice1000** [UniMath/UniMath#1877] [Q&A] [answered]
Just want to confirm because I didn't find anything by searching. The result has been proved in

```bibtex
@article{S1S1EqTorus,
  title = {The equivalence of the torus and the product of two circles in homotopy type theory},
  author = {Kristina Sojakova},
  year = 2015,
  journal = {CoRR},
  volume = {abs/1510.03918},
  url = {http://arxiv.org/abs/1510.03918},
  eprinttype = {arXiv},
  eprint = {1510.03918},
  timestamp = {Mon, 13 Aug 2018 16:49:09 +0200},
  biburl = {https://dblp.org/rec/journals/corr/Sojakova15.bib},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
```

**Comment by nmvdw** [comment, marked as answer]
As far as I know, there is no such result in UniMath (or even in a repository outside of UniMath that usees UniMath). The paper by Sojakova doesn't mention a formalization either (https://dl.acm.org/doi/abs/10.1145/2992783). Was it even formalized? The paper also neither cites CoqHoTT nor UniMath

**Reply by ice1000** [reply]
Thank you for the fast answer!
